### ISSUE TITLE
No `aria-label` attribute in <input> field

### ISSUE THREAD

**Issue by zzAlexzz** [haoxins/react-flatpickr#85]
Input doesn't have the `aria-label` attribute for passing accessibility, I am wondering how can I set `aria-label` value in second `<input>` tag?
![image](https://user-images.githubusercontent.com/31755924/53776118-c5c52800-3f40-11e9-94ad-69916d217c3a.png)

![image](https://user-images.githubusercontent.com/31755924/53776156-edb48b80-3f40-11e9-8d8d-720bbc5b26b3.png)

Codes:
```javascript
import Flatpickr  from 'react-flatpickr';

...
        <Flatpickr
            aria-label="Date of Birth"
            data-alt-input-class="form-control form-control-icon-calendar form-control-datepicker"
            options={{
                altInput: true,
                altFormat: 'j M Y',
                dateFormat: 'Y-m-d',
                wrap: true,
                allowInput: true,
                parseDate: (date) => {
                    const formats = ['DD/MM/YYYY', 'D MMM YYYY', 'DD-MM-YYYY', 'YYYY-MM-DD'];

                    // @ts-ignore
                    moment.parseTwoDigitYear = (input) => {
                        const x = Number(input);
                        const currentYear = Number(
                            new Date().getFullYear().toString().substr(-2),
                        );
                        return x + (x > currentYear ? 1900 : 2000);
                    };

                    const parsedDate = moment(date, formats);
                    if (parsedDate.isValid()) return parsedDate.toDate();
                    return new Date();
                },
            }}
        >
            <input
                name="date_of_birth"
                aria-label="Date of Birth"
                placeholder="Please enter your date of birth"
                type="date"
                data-input
            />
        </Flatpickr>
...
```
related ticket in flatpickr:
https://github.com/flatpickr/flatpickr/issues/1269

**Comment by jacobmischka** [comment]
Dupe of #80, run `npm upgrade` in your project and you'll get new flatpickr.

**Comment by zzAlexzz** [comment]
Thanks for the replying. :-o)
I just found the solution:
in `options`, pass method `onReady: () => {}` to add attribute  `aria-label`.
```javascript
import Flatpickr  from 'react-flatpickr';

...
        <Flatpickr
            aria-label="Date of Birth"
            data-alt-input-class="form-control form-control-icon-calendar form-control-datepicker"
            options={{
                altInput: true,
                altFormat: 'j M Y',
                dateFormat: 'Y-m-d',
                wrap: true,
                allowInput: true,
                parseDate: (date) => {
                    const formats = ['DD/MM/YYYY', 'D MMM YYYY', 'DD-MM-YYYY', 'YYYY-MM-DD'];

                    // @ts-ignore
                    moment.parseTwoDigitYear = (input) => {
                        const x = Number(input);
                        const currentYear = Number(
                            new Date().getFullYear().toString().substr(-2),
                        );
                        return x + (x > currentYear ? 1900 : 2000);
                    };

                    const parsedDate = moment(date, formats);
                    if (parsedDate.isValid()) return parsedDate.toDate();
                    return new Date();
                },
                onReady: (a, b, fp) => {
                    fp.altInput!.setAttribute('aria-label', props.label);
                },
            }}
        >
            <input
                name="date_of_birth"
                aria-label="Date of Birth"
                placeholder="Please enter your date of birth"
                type="date"
                data-input
            />
        </Flatpickr>
...
```
